src/utils/create-utils.ts

```diff
@@ -151,3 +158,42 @@ export const createComponentDetectionQuery = async (
 
   return cdq.status.componentDetected;
 };
+
+/**
+ * Create SPIAccessTokenBinding CR
+ *
+ * @param url the URL of the git repository
+ * @param namespace namespace to create the binding
+ * @returns Returns created SPIAccessTokenBinding resource
+ */
+export const initiateAccessTokenBinding = async (url: string, namespace: string) => {
+  const name = uniqueId(`hac-dev-import-${uid()}`);
+  const requestData = {
+    apiVersion: `${SPIAccessTokenBindingModel.apiGroup}/${SPIAccessTokenBindingModel.apiVersion}`,
+    kind: SPIAccessTokenBindingModel.kind,
+    metadata: {
+      name,
+      namespace,
+    },
+    spec: {
+      repoUrl: url,
+      permissions: {
+        required: [
+          { type: 'r', area: 'repository' },
+          { type: 'w', area: 'repository' },
+        ],
+      },
+      secret: {
+        name: 'hac-dev-repo-token',
```

Comment:
Won't this secret end up clashing with other components that require auth?

Reply:
Appended uid to secret name.